OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageAction.java

```diff
@@ -125,6 +133,11 @@ public String getClickName() {
         return clickName;
     }
 
+    @NonNull
```

Comment:
this is marked as notNull but 

`pageId = json.optString(PAGE_ID, null);` it could end being null, maybe set a default value? tr change to Nullable

Reply:
fixed!